source/includes/_snapshots.md

```diff
@@ -1 +1,126 @@
 # Snapshots
+
+## The Snapshot Resource
+
+### Attributes:
+
+
+Attribute | Type | Description
+--------- | ------- | -------
+`id` | string | A unique identifier of this snapshot.
+`created_at` | datetime | The time the snapshot was uploaded to or created on the manager.
+`status` | string | Status of this snapshot - created/failed/uploading/uploaded.
+`error` | string | Message of an error if snapshot creation failed.
+
+## List Snapshots
+`GET /api/v2/snapshots`
+
+Lists all snapshots
+
+> Request Example
+
+```shell
+$ curl -XGET http://localhost/api/v2/snapshots
+```
+
+> Response Example
+
+```json
+{
+    "items": [
+        {
+            "created_at": "2015-12-04 13:34:45.080009",
+            "error": "",
+            "id": "snapshot1",
+            "status": "created"
+        },
+        {
+            "created_at": "2015-12-04 13:35:04.972249",
+            "error": "",
+            "id": "snapshot2",
+            "status": "created"
+        }
+    ],
+    "metadata": {
+        "pagination": {
+            "offset": 0,
+            "size": 10000,
+            "total": 2
+        }
+    }
+}
+```
+
+### Response
+
+Field | Type | Description
+--------- | ------- | -------
+`items` | list | A list of `Snapshot` resources.
+`metadata` | object | A dictionary containing data useful for pagination logic.
+
+
+## Create Snapshot
+`PUT /api/v2/snapshots/{snapshot-id}`
+
+Creates a new snapshot.
+
+### URI Parameters
+* snapshot-id: The id of the new snapshot
+
+### Request Body
+Property | Default | Description
+--------- | ------- | -----------
+`include_metrics` | False | Should InfluxDB data be included in the snapshot.
+`include_credentials` | False | Should agent keys, even those specified in uploaded blueprints, be included in the snapshot.
+
+### Response
+An `Execution` resource representing the create snapshot workflow execution.
+
+
+## Delete Snapshot
+`DELETE /api/v2/snapshots/{snapshot-id}`
+
+Deletes an existing snapshot
+
+### URI Parameters
+* snapshot-id: The id of the snapshot to be deleted
+
+### Response
+An empty `Snapshot` resource, with one non-empty field (its id).
+
+## Restore Snapshot
+`POST /api/v2/snapshots/{snapshot-id}/restore`
+
+### URI Parameters
+* snapshot-id: The id of the snapshot to be restored
+
+### Request Body
+Property | Default | Description
+---------|---------|-------------
+`force`  |  False  | Force restoring the snapshot on a manager that already contains blueprints and/or deployments
+`recreate_deployments_envs` | True | Should deploments environments for restored deployments be created - this is rather natural and strongly recommended.
```

Comment:
`true`

"Specifies whether deployment environments should be created for restored deployments."